lbrynet/core/Wallet.py

```diff
@@ -130,54 +236,161 @@ def create_tables(transaction):
                                 "    name text, " +
                                 "    txid text, " +
                                 "    n integer)")
+            transaction.execute("CREATE TABLE IF NOT EXISTS claim_cache (" +
+                                "    row_id INTEGER PRIMARY KEY AUTOINCREMENT, " +
+                                "    claim_id TEXT UNIQUE NOT NULL, " +
```

Comment:
is claim_id actually unique here? shouldn't it be something like outpoint? or is this the cache for winning claims?